frontend/Dockerfile

```diff
@@ -0,0 +1,22 @@
+# python:3.6-alpine3.9 specific working version
+FROM python@sha256:ad2b24eacc9865244d7b1ae198d507f4ce8f506484c35d27158e7f1dd6041a8b
```

Comment:
Frontend does not seem to be using Python, so maybe just switch to the alpine 3.9 image or such?